src/game.ts

```diff
@@ -76,43 +61,17 @@ export class Game {
     requestAnimationFrame(update);
   }
 
-  private drawPlayerHealth() {
-    if (this.ctx === null) {
-      throw new Error('CanvasRenderingContext2D is null.');
-    }
-
-    // Set the font color to black (or any color that contrasts with the background)
-    this.ctx.fillStyle = 'black';
-    this.ctx.font = '18px Arial';
-    this.ctx.fillText(`Health: ${this.player.getHealth()}`, 10, 30);
-  }
-
   update() {
     // Update game logic here
-    playerInput(this.keysPressed, this.player);
-    updateCharacters(this.player, this.enemies);
+    const player = this.characterManager.getPlayer();
+    playerInput(this.keysPressed, player);
```

Comment:
Just inline `this.characterManager.getPlayer()` here.

Reply:
done f6376b4dba2108453bab81ab34afadf7c9e9d207